.github/workflows/integration-parquet.yml

```
@@ -46,7 +46,7 @@ jobs:
           python -m venv venv
           source venv/bin/activate
           pip install --upgrade pip
-          pip install pyarrow pyspark
+          pip install pyarrow==6.0.1 pyspark==3.2.0
```

I left it open on purpose - to ensure we remain compatible.

Maybe then using 

```suggestion
          pip install pyarrow==6 pyspark==3
```